astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
@bmorris3 – Can you please at least point out the weird argument order  in the `AirmassWhatever` docstring and explain with one or two sentences or examples why it's different from `AltitudeRange` (which has `min` as first parameter).